.stylelintignore

```diff
@@ -2,3 +2,28 @@
 **/_site/**/*.css
 packages/cli/test/functional/**/expected/**/*.css
 packages/vue-components/**/*.css
+
+# --- packages/vue-components .vue files ---
+
+packages/vue-components/src/Navbar.vue
```

Comment:
meant to shift all of stylelint to another PR, so you wouldn't have to write this huge ignore list here 😅

but am fine with this as well.

Reply:
> Enable linting for dropdown.vue

let's update the message body to highlight vue css lint enabling as well

Reply:
Thanks @ang-zeyu. Updated the message body.